app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionOverlay.kt

```diff
@@ -49,8 +51,25 @@ class TrackingProtectionOverlay(
 
     @Suppress("MagicNumber", "InflateParams")
     private fun showTrackingProtectionOnboarding() {
+
         if (!getToolbar().hasWindowFocus()) return
 
+        when (getToolbar().parent) {
```

Comment:
One slight improvement regarding knowing if the toolbar is at the top or at the bottom since the current check is pretty frail: What we usually use and something that would also increase readability is 
> when (settings.shouldUseBottomToolbar && settings.isDynamicToolbarEnabled)